services/analyzer/sequelize-default-value.js

```diff
@@ -12,7 +12,7 @@ class DefaultValueExpression {
   }
 
   parse() {
-    if (this.expression === null || this.expression === undefined) {
+    if (!this.expression) {
```

Comment:
This was fine before 😅 , this will transform empty string to null which we do not want 😉

Reply:
Oops. I did that because the linter was complaining over code complexity. I'll revert the change.